addons/web/static/src/views/fields/copy_clipboard/copy_clipboard_field.js

```diff
@@ -20,19 +20,22 @@ class CopyClipboardField extends Component {
         return `o_btn_${this.props.type}_copy btn-sm`;
     }
     get fieldProps() {
-        const fieldProps = {...this.props};
+        const fieldProps = { ...this.props };
+        delete fieldProps.disabled;
         delete fieldProps.string;
         return fieldProps;
```

Comment:
Could you please use this instead? :
```
return omit(this.props, [
    "disabled",
    "string"
]);
```
And don't forget to add ` import { omit } from "@web/core/utils/objects"`